lib/openfeature/sdk/client.rb

```diff
@@ -1,18 +1,83 @@
 # frozen_string_literal: true
 
+require "concurrent"
+require "forwardable"
+
 module OpenFeature
   module SDK
     # TODO: Write documentation
     #
     class Client
+      extend Forwardable
```

Comment:
I'm not seeing anything using this yet 🤔

Reply:
Good call. I've removed it